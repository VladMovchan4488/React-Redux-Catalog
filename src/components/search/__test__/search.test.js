import React from "react";
import Search from "../search";
import Adapter from "enzyme-adapter-react-16";
import { shallow, configure } from "enzyme";
import toJson from "enzyme-to-json";

configure({ adapter: new Adapter() });

it("Snaphot List", () => {
  const props = {
    match: { params: { products: "" } },
    history: {
      location
    },
    getSearchValue() {},
    input: ""
  };

  const wrapper = shallow(<Search.WrappedComponent {...props} />);

  expect(toJson(wrapper)).toMatchSnapshot();
});
