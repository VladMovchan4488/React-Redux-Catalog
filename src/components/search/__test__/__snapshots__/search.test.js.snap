// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
<styled.div>
  <styled.input
    className="form-control form-control-sm ml-3 w-75"
    onChange={[Function]}
    onInput={[Function]}
    placeholder="LiveSearch by name in category"
    value=""
  />
</styled.div>
`;

exports[`Snaphot List 1`] = `
<styled.div>
  <styled.input
    className="form-control form-control-sm ml-3 w-75"
    onChange={[Function]}
    onInput={[Function]}
    placeholder="LiveSearch by name in category"
    value=""
  />
</styled.div>
`;
